### PR TITLE
Fix string interpolation typo.

### PR DESCRIPTION
**Pull request by neonichu**

### CODE REVIEW

File: Source/CarthageKit/Xcode.swift [hunk 1]
```diff
@@ -332,7 +332,7 @@ public enum SDK: String {
 
 	/// Attempts to parse an SDK name from a string returned from `xcodebuild`.
 	public static func fromString(string: String) -> Result<SDK, CarthageError> {
-		return Result(self(rawValue: string), failWith: .ParseError(description: "unexpected SDK key \"(string)\""))
+		return Result(self(rawValue: string), failWith: .ParseError(description: "unexpected SDK key \(string)"))
```
Comment (ikesyo) [inline Source/CarthageKit/Xcode.swift]: I think this intends to quote the `string` value, so we should fix as `"unexpected SDK key \"\(string)\""`.

**Comment by ikesyo** [comment]
Nice catch! :sparkles: Additionally, there is  the same miss in `ProductType.static.fromString()`. Could you fix that too, please? :pray:

**Comment by ikesyo** [comment]
@neonichu I did the remaining work in #651 and merged it. Thank you again! :grin:

**Comment by neonichu** [comment]
Thank you, sorry for having forgotten this.

**Pull request merged**